core/src/main/scala/kafka/network/RequestChannel.scala

```diff
@@ -73,14 +73,14 @@ object RequestChannel extends Logging {
     buffer = null
     private val requestLogger = Logger.getLogger("kafka.request.logger")
 
-    private def requestDesc: String = {
+    private def requestDesc(details: Boolean): String = {
```

Comment:
Thanks for the review. However, I think number of times a method is called with particular param value is not a good argument for deciding default value of a param.